pytest-saas/doc/changes/unreleased.md

```diff
@@ -1 +1,9 @@
 # Unreleased
+
+## Feature
+
+* #18: Implemented keep alive for saas-database
+
+## Bugfixes
+
+* #19: Fixed removing ip whitelist rule at the end of the test session
```

Comment:
If the PR fixes both issues (#18 and #19), add the comment in the PR

Reply:
Thanks for this important finding. 
I added closes comment to PR
Please approve again